src/components/EmailSubscription/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react"
+import "./index.scss"
+import { ReactComponent as RightArrow } from "images/icons/rightArrow.svg"
+import { subscribeMailchimp } from "utils/Api"
+
+const SubmissionStates = {
+  NotSubmitted: 0,
+  Loading: 1,
+  Submitted: 2,
+  Errored: 3,
+}
+
+export const EmailSubscription: React.FC = () => {
+  const [email, setEmail] = useState("")
+  const [state, setState] = useState(SubmissionStates.NotSubmitted)
+  const [message, setMessage] = useState("")
+
+  const submitEmail = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault()
+    setState(SubmissionStates.Loading)
+    setMessage("Please wait while your message is being submitted...")
```

Comment:
One thing, I just remembered really late but can we validate that the form data is not an empty string and is valid email [regex](https://www.w3resource.com/javascript/form/email-validation.php).

Reply:
Will do!

Reply:
This has been added, but will probably be best to test with #45.